apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -0,0 +1,81 @@
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import React, {useCallback} from 'react';
+import DCDO from '@cdo/apps/dcdo';
+import SectionProgress from '../sectionProgress/SectionProgress';
+import Button from '@cdo/apps/templates/Button';
+import {setShowProgressTableV2} from '@cdo/apps/templates/currentUserRedux';
+import SectionProgressV2 from './SectionProgressV2';
+import UserPreferences from '@cdo/apps/lib/util/UserPreferences';
+
+function SectionProgressSelector({
+  showProgressTableV2,
+  setShowProgressTableV2,
+}) {
+  const onShowProgressTableV2Change = useCallback(
+    e => {
+      const shouldShowV2 = !showProgressTableV2;
+      new UserPreferences().setSortByFamilyName(shouldShowV2);
```

Comment:
Why does this include `.setSortByFamilyName`?

Reply:
A copy paste mistake, good catch